src/helpers/index.ts

```diff
@@ -107,3 +109,24 @@ export function splitArray<Type>(array: Type[], chunkSize: number): Type[][] {
     array.slice(index * chunkSize, (index + 1) * chunkSize)
   )
 }
+
+export function getSupportedChainIds(): ChainId[] {
+  return (DEFAULT_CHAIN_ID || '')
+    .split(',')
+    .filter(Boolean)
+    .map((chainId) => Number(chainId))
+}
+
+export function getEnvironmentChainId() {
+  const chainId = getSupportedChainIds()[0]
+  switch (chainId) {
+    case ChainId.ETHEREUM_MAINNET.valueOf():
+      return ChainId.ETHEREUM_MAINNET
+    case ChainId.ETHEREUM_GOERLI:
```

Comment:
I know you didn't change this, but I wonder why do we need the valueOf for Mainnet and not for the other chains...?